stable/tomcat/README.md

```diff
@@ -0,0 +1,68 @@
+# Tomcat
+
+[Tomcat](http://tomcat.apache.org) is is an open source implementation of the Java Servlet, JavaServer Pages, Java Expression Language and Java WebSocket technologies.
+
+## Introduction
+
+This chart creates a [tomcat application server](http://tomcat.apache.org) Deployment, plus http Services for the server.
+The chart offers an optimization for application updates running in a servlet container-type engines like tomcat and [Jboss](http://jbossas.jboss.org). The chart uses the WAR, EAR, and other deployable components outside of the Servlet engine as sidecar container so application upgrades requires the sidecar container image only to be updated and not the Servlet engine as if both would run at the same image. 
+
+
+## Prerequisites
+
+- Kubernetes 1.7+ with Beta APIs enabled
+
+## Provider-specific Prerequisites
+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/tomcat
+```
+
+This command deploys a tomcat dedicated server with sane defaults.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
```

Comment:
Please add a table to explain the configurable components. Thanks. Please refer https://github.com/kubernetes/charts/blob/master/stable/bitcoind/README.md.

Reply:
Added